fsm/handler.go

```diff
@@ -166,7 +166,7 @@ func HandleStateActiveEventPduSessRelease(event SmEvent, eventData *SmEventData)
 	smCtxt := txn.Ctxt.(*smf_context.SMContext)
 
 	if err := producer.HandlePDUSessionSMContextRelease(eventData.Txn); err != nil {
-		stats.PublishMsgEvent(mi.Smf_msg_type_pdu_sess_release_req_failure)
+		stats.PublishMsgEvent(mi.Smf_msg_type_pdu_sess_release_rsp_failure)
```

Comment:
req failiure was correct right?

Reply:
Please see my reply to your other comment